man/man5/zpool-features.5

```diff
@@ -143,6 +143,70 @@ read\-only mode.
 Some features depend on other features being enabled in order to function
 properly. Enabling a feature will automatically enable any features it
 depends on.
+
+.SS "Compatibility feature sets"
```

Comment:
As above, most of the macros in this section aren't working. I don't know why this is - some enclosing block inhibiting them? Help please...

Reply:
Looks like zpool-features.5 isn't well formatted as a man page? Was it generated from some parent documentation somehow? Possibly not too much work to convert to standard mdoc, but any context would be welcome...

Reply:
Possible that some pages, like zpool-features.5, are formatted using one man page standard, while others, like zpoolprops.8, are formatted to a different one. Fun times! I will try to reformat this section of zpool-features to match the document format.

Reply:
> Possible that some pages, like zpool-features.5, are formatted using one man page standard, while others, like zpoolprops.8, are formatted to a different one.

mdoc vs groff_man. Time to reference [Standards](https://xkcd.com/927/)